pulp_2to3_migration/app/tasks/reset.py

```diff
@@ -33,6 +34,7 @@ def reset_pulp3_data(migration_plan_pk):
                    total=len(pulp2_plugins))
     with ProgressReport(**pb_data) as pb:
         for plugin in plan.get_plugin_plans():
+            RepoSetup.reset_plugin(plugin.type)
```

Comment:
I haven't added any tests for the reset. I'm not sure what to test apart from what we've already had for the reset feature.